cassandra/src/test/java/org/apache/calcite/test/CassandraAdapterTest.java

```diff
@@ -100,8 +100,8 @@ static void load(Session session) {
         .returns("tweet_id=f3c329de-d05b-11e5-b58b-90e2ba530b12\n"
                + "tweet_id=f3dbb03a-d05b-11e5-b58b-90e2ba530b12\n")
         .explainContains("PLAN=CassandraToEnumerableConverter\n"
-                + "  CassandraLimit(fetch=[2])\n"
-                + "    CassandraProject(tweet_id=[$2])\n"
+                + "  CassandraProject(tweet_id=[$2])\n"
+                + "    CassandraLimit(fetch=[2])\n"
                 + "      CassandraFilter(condition=[=($0, '!PUBLIC!')])\n");
   }
```

Comment:
Does the same cost lead to the change?

Reply:
yes, the costs are the same. However, I do think in most case the new plan is better.